src/publishers/EnviroDIYPublisher.cpp

```diff
@@ -140,19 +228,37 @@ int16_t EnviroDIYPublisher::publishData(Client* outClient) {
         txBufferAppend(_baseLogger->getSamplingFeatureUUID());
 
         txBufferAppend(timestampTag);
-        txBufferAppend(
-            Logger::formatDateTime_ISO8601(Logger::markedLocalEpochTime)
-                .c_str());
-        txBufferAppend('"');
+
```

Comment:
Can this be written to only use the square bracket arrays if there is more than one data point to send? That would make it semi-backward-compatible with MonitorMyWatershed.

Reply:
It would be pretty straightforward but I'm not sure of the value. That means that if `sendEveryX > 1` or a data point ever fails to send, then arrays will start being used and no future transmission will ever work because the site will reject all of them.

That seems like a giant footgun but it might be helpful for the initial testing. If you want it to test right now before the site updates I would encourage you to just edit the code to remove the square brackets.

Reply:
Point taken.  I'll set this up just for my temporary testing.